exporters/opensearch-exporter/README.md

```diff
@@ -16,165 +16,142 @@ record value type (see the value type in the Zeebe protocol). Each of these inde
 corresponding pre-defined mapping to facilitate data ingestion for your own ETL jobs. You can find
 those as templates in this module's resources folder.
 
-> **Note:** the indexes are created as required, and will not be created twice if they already exist. However,
-> once disabled, they will not be deleted, that is up to the administrator. Similarly, data is never deleted by
+> **Note:** the indexes are created as required, and will not be created twice if they already
+> exist. However,
+> once disabled, they will not be deleted, that is up to the administrator. Similarly, data is never
+> deleted by
 > the exporter, and must, again, be deleted by the administrator when it is safe to do so.
 
-## Usage
+## Configuration
 
 > **Note:** As the exporter is packaged with Zeebe, it is not necessary to specify a `jarPath`.
 
-You can configure the Elasticsearch Exporter with the following arguments:
+The exported can be enabled by configuring it with the classpath in the Broker settings.
+For example:
+
+```yaml
+exporters:
+  opensearch:
+    className: io.camunda.zeebe.exporter.opensearch.OpensearchExporter
+    args:
+    # Please refer to the table below for the available args options
+```
+
+The exporter can be configured by providing `args`. The table below explains all the different
+options, and the default values for these options.
+
+|      Option      |                                       Description                                       |         Default         |
+|------------------|-----------------------------------------------------------------------------------------|-------------------------|
+| url              | Valid URLs as comma-separated string                                                    | `http://localhost:9200` |
+| requestTimeoutMs | Request timeout (in ms) for the Opensearch client                                       | `30000`                 |
```

Comment:
I noticed that this arg was undocumented previously. So, I looked around, if this is even configurable. I could not find any tests showing this configuration can be set. I also noticed that the `requestTimeoutMs` is not part of the OpensearchExporterConfiguration's toString method. 

❓ Did you happen to verify whether this option can be configured?

Reply:
Seems it was added because of failing ITs. But as it is possible to configure I still vouch for adding it here.